x-pack/legacy/plugins/alerting/server/types.ts

```diff
@@ -95,4 +95,4 @@ export interface AlertingPlugin {
   start: PluginStartContract;
 }
 
-export type AlertTypeRegistry = PublicMethodsOf<AlertTypeRegistry>;
+export type AlertTypeRegistry = PublicMethodsOf<OrigAlertTypeRegistry>;
```

Comment:
:information_source: The usual 3.7 issue, that you cannot have an import and export with the same name (but different values) in one file anymore.